README.md

```diff
@@ -0,0 +1,10 @@
+
+
+Mining Server Proxy
+=====================================
+
+[![Build Status](https://travis-ci.org/TheBlueMatt/mining-proxy.svg?branch=master)](https://travis-ci.org/TheBlueMatt/mining-proxy)
+
+A simple proxy which supports serving work via the Stratum and the [BetterHash](https://github.com/TheBlueMatt/bips/blob/master/bip-XXXX.mediawiki) mining protocols.
+
+Work can be requested from bitcoind using the [2018-02-miningserver](https://github.com/TheBlueMatt/bitcoin/commits/2018-02-miningserver) patchset. Payout information is also available in a similar manner.
```

Comment:
You lost information (about the protocl used). Also note that pool protocol isnt to bitcoind, its to the pool daemon in this repo.